xml/System.Reflection.Metadata.Ecma335/InstructionEncoder.xml

```diff
@@ -28,8 +28,20 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Encodes instructions.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Encodes Common Intermediate Language (CIL) instructions.</summary>
+    <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+## Remarks
+The <xref:System.Reflection.Metadata.Ecma335.InstructionEncoder> class is used to emit CIL instructions that make up a method body. For a complete example of emitting a method, see the <xref:System.Reflection.Metadata.Ecma335.MetadataBuilder> class remarks.
```

Comment:
```suggestion
The <xref:System.Reflection.Metadata.Ecma335.InstructionEncoder> class is used to emit CIL instructions that make up a method body. For a complete example of emitting a method, see the <xref:System.Reflection.Metadata.Ecma335.MetadataBuilder> class example.
```

Reply:
"For a complete example ... , see ... example" reads odd, i'll change to documentation instead.

Reply:
That sounds better, thanks!